client/src/app/TabsProvider.js

```diff
@@ -17,6 +17,7 @@ import {
 import replaceIds from '@bpmn-io/replace-ids';
 
 import { Linter as BpmnLinter } from '@camunda/linting/lib/Linter';
+import { FormLinter } from '@camunda/form-linting/lib/FormLinter';
```

Comment:
Can't we simply use 

`import { FormLinter } from '@camunda/form-linting';`

as suggested by the README?

Reply:
I see we're doing the same for the BPMN linter though ... so it might be okay.